src/Equinor.ProCoSys.IPO.Infrastructure/IPOContext.cs

```diff
@@ -32,9 +32,9 @@ public class IPOContext : DbContext, IUnitOfWork, IReadOnlyContext
             _currentUserProvider = currentUserProvider;
         }
 
-        // enable these lines to log sql queries locally
-        //protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
-        //    => optionsBuilder.LogTo(System.Console.WriteLine);
+        //enable these lines to log sql queries locally
```

Comment:
I guess optionsBuilder.LogTo(System.Console.WriteLine) is safe to use in any environemnt, so remove comment on line 35